### ISSUE TITLE
Up-mixing errors with new installs, but not with old copies

### ISSUE THREAD

**Issue by bthj** [ircam-ismm/node-web-audio-api#29]
Hi,
For audio graphs that worked before, I'm now encountering the error:
```
thread '<unnamed>' panicked at 'Up-mixing from 1 to 3 channels not supported', /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/web-audio-api-0.28.0/src/render/quantum.rs:509:22
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Rendering failed: Error: Up-mixing from 1 to 3 channels not supported
```

Maybe this is related to issue #23, at least there is also mention of up-mixing issues.

In one project, which I haven't updated recently, rendering those same audio graphs works fine. That project references v0.6.0 of node-web-audio-api. If I create a new project referencing v0.6.0 as well, rendering those graphs produces the errors. If I copy in `node_modules` from the project which hasn't been updated, then rendering works fine again. Trying to identify differences between that old `node_modules` tree and the fresh install (referencing v0.6.0), I do see some: 
- old:
```
├─┬ node-web-audio-api@0.6.0
│ ├── @napi-rs/cli@2.14.6
│ └─┬ @node-rs/helper@1.3.3
│   └── @napi-rs/triples@1.1.0
```
- fresh:
```
└─┬ node-web-audio-api@0.6.0
  ├── @napi-rs/cli@2.16.2
  └─┬ @node-rs/helper@1.3.3
    └── @napi-rs/triples@1.1.0
```

Copying only those dependencies from the old `node_modules` tree does not solve the issue, but copying everything (node_modules/*) does solve it. So some other dependency seems to be the culprit, which I haven't been able to identify. Does any possibility come to mind?

I'll attach a project to reproduce this - [kromosynth-render-test.zip](https://github.com/ircam-ismm/node-web-audio-api/files/12267424/kromosynth-render-test.zip) - in the unzipped folder:
```
npm i
node server.js './testGenome1.json'
```
should produce the error (also when referencing files ...2.json and ...3.json), but this should work:
```
node server.js './testGenome4.json'
```

All files should render after copying in the old [node_modules](https://uio-my.sharepoint.com/:u:/g/personal/bthj_uio_no/EalvdY7x2ltKquFIXZSE12sB0PFSuaeabzHz13mT_Xnc3Q?e=kDyllp).

The errors also occur if I reference the latest version, v0.11.0.

**Comment by b-ma** [comment]
Hey, 

Actually having the error `'Up-mixing from 1 to 3 channels not supported'` seems quite normal to me because it is not supported and has never been, cf.:
- https://webaudio.github.io/web-audio-api/#channel-up-mixing-and-down-mixing
- https://github.com/orottier/web-audio-api-rs/blob/main/src/render/quantum.rs#L303

What may explain this change in your app is that since v0.11, the full `AudioDestination` API is implemented, i.e. before that version using `destination.channelCount` didn't work (cf. https://github.com/ircam-ismm/node-web-audio-api/issues/21). So I think that your code was not really working as expected and now it fails as it should.

Maybe this hides another issue (this `node_modules` thing is quite weird), but first it should be fixed at your library level: if you use fancy number of channels (e.g. 3) you should explicitly use `channelInterpretation = 'discrete'`, do the channel 
routing yourself and don't rely on the default multichannel mixing rules. 

I added an example of how to do that https://github.com/ircam-ismm/node-web-audio-api/blob/main/examples/multichannel.mjs, maybe this can help

**Comment by bthj** [comment]
I'm not explicitly changing channels from 1 to 3. In the [fork of virtual-audio-graph](https://github.com/synth-is/virtual-audio-graph) I'm using, I first tried adding a condition to set `channelInterpretation = 'discrete'` where ChannelMerger nodes are handled (for now just in the node_modules dist), without luck, but then I quickly set that property everywhere I found any node or audio context being handled, and that got rid of the issue. I'll have to go through this again with a bit more thought on exactly where it matters, but this definitely helped, thanks!
